packages/terra-collapsible-menu-view/src/CollapsibleMenuView.jsx

```diff
@@ -28,6 +29,15 @@ const propTypes = {
    * Bounding container for the menu, will use window if no value provided.
    */
   boundingRef: PropTypes.func,
+
+  /**
+   * Sets the more button variant. One of `neutral`, `utility`.
+   */
+  moreButtonVariant: PropTypes.oneOf([Button.Opts.Variants.NEUTRAL, Button.Opts.Variants.UTILITY]),
```

Comment:
thoughts on changing propname to `menuButtonVariant` since it discloses the menu? 

Thinking it will match better to `menuWidth` and other menu naming already used in collapsible-menu-view,
Could even also swap props order so props table reads: children, boundingRef, menuWidth, menuButtonVariant

Reply:
That sounds good. Changed it: https://github.com/cerner/terra-framework/pull/943/commits/7e50238a5d34187af5940b5a9e9c306be63399d6